elastio-s3-changelog/README.md

```diff
@@ -11,11 +11,18 @@ Then, the Elastio `iscan` job reads those events to perform the scanning of new
 
 1. First, you need to enable Amazon EventBridge for your S3 buckets by following these instructions:
     [Enabling Amazon EventBridge](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-event-notifications-eventbridge.html).
-2. You can use
-    [this quick-create link](https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v1/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
-    to create the stack.
-
-    > You might need to switch to the region where your Elastio CloudFormation stack is deployed.
+2. Use one of the following quick-create links. Choose the region where your Elastio CloudFormation stack is deployed:
+    * [ap-south-1](https://ap-south-1.console.aws.amazon.com/cloudformation/home?region=ap-south-1#/stacks/create/review?templateURL=https://elastio-prod-artifacts-us-east-2.s3.us-east-2.amazonaws.com/contrib/elastio-s3-changelog/v2/cloudformation-multiple-buckets.yaml&stackName=elastio-s3-changelog)
```

Comment:
All the links are broken. How did they end up leading to v2 version?

Reply:
Because I changed the version.

Reply:
I see, I missed that small change in version file. Changing the version isn't needed for several reasons:
- This CFN template hasn't been released and used in prod yet, so we can do any breaking changes until that happens without bumping versions
- The version here isn't intended to be bumped basically ever. It is there only just to account for the case when we do a breaking change. If you want to add an indicator of a patch number, then let's just add `Patch: N` output to the outputs of the single-bucket CFN stack. This may be useful to discover which exact version the user used.